server/WebAPI/DTO/TicketDto.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Entities.Models;
+using System;
+using System.Collections.Generic;
+
+namespace WebAPI.DTO
+{
+    public class TicketDto
+    {
+        public int Id { get; set; }
+        public int UserId { get; set; }
+        public string Price { get; set; }
```

Comment:
PP: price and currency separately

**PP:** This is a **P**ersonal **P**reference. The person reviewing thinks it’s better to change/fix this. The review will pass.